### PR TITLE
[HDRP][VT] Bugfix multiple resolvers

### PR DESCRIPTION
**Pull request by debaetsd**
### Checklist for PR maker
- [  ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [x] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
This PR fixes an issue that caused VT to stop requesting pages when multiple cameras where (offscreen) rendering.

---
### Testing status
Added additional tests to HDRP_RuntimeTests to verify this exact senario
These also check if VT plays nicely along with the global mip bias (required for DLSS et al)

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs [hunk 1]
```diff
@@ -244,7 +244,7 @@ public partial class HDRenderPipeline
 #if ENABLE_VIRTUALTEXTURES
                     // Note: This pass rely on availability of vtFeedbackBuffer buffer (i.e it need to be write before we read it here)
                     // We don't write it when doing debug mode, FullScreenDebug mode or path tracer. Thus why this pass is call here.
-                    m_VtBufferManager.Resolve(m_RenderGraph, hdCamera, vtFeedbackBuffer);
+                    hdCamera.ResolveVirtualTextureFeedback(m_RenderGraph, vtFeedbackBuffer);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs:247]: Question what is happening here with multiple camera? isn't the VTFeedback PR overwrite by each camera result? or there is a process on CPU side in PushFullScreenVTFeedbackDebugTexture which do it, mean that there is some sort of sync point between each camera?

**Comment by remi-chapelain** [comment]
Hey, do you have in mind what could need further testing in the context of the fix ?
Thanks ! :)

**Review by FrancescoC-unity — APPROVED** [review]
This looks good to me, though I am curious to what the answer is to seb's question.

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs (hunk 1 above)
Comment (debaetsd) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs:247]: This is exactly the change :)
Before this PR that VTBufferManager was part of the HDRenderPipeline but now it is owned by HDCamera. So now each camera has a (CPU, async readback) buffer rather then 1 shared across all cameras. 

This used to be working so I guess this broke recently (maybe with the introduction of RG?)
I added an explicit test for this in HDRP_RuntimeTests (where the other VT tests also live)

**Comment by debaetsd** [comment]
> Hey, do you have in mind what could need further testing in the context of the fix ?
> Thanks ! :)

I modifies HDRP_RuntimeTests (003-VirtualTexturing) to explicitly test multiple cameras so not a whole lot needs to be tested imho. If the existing VT tests keep working & those new ones also pass, I'm fairly confident about it.

**Comment by debaetsd** [comment]
@kecho This PR will fix the issues you were seeing wrt the global mipbias. The test I added is almost a copy of your previous setup (multiple off-screen VT quads rendered with various global mip bias applied).

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
note: launched yamato with a custom revision that is suppose to be green for check: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/HDRP%252Fvt-bugfix-resolver/.yamato%252Fall-hdrp.yml%2523Nightly_HDRP_CUSTOM-REVISION/7060530/job/pipeline

**Review by JulienIgnace-Unity — APPROVED** [review]

**Comment by debaetsd** [comment]
@sebastienlagarde Well that yamato builder cannot be green as I haven't updated the reference images yet ( not sure what is happening but the builder that should have invalidated them reports everything green.. investigating atm)

**Review by TomasKiniulis — APPROVED** [review]
Seems good. Tried creating new VT, checked with multiple Scene/Game views with multiple cameras in Editor and Playmode didn't encounter any PR related issues.

**Comment by GraphineDieter** [comment]
@sebastienlagarde Updated reference images. PR is good to go

**Comment by sebastienlagarde** [comment]
@GraphineDieter  should this PR be backported? can you create the backport pr for 21.1 and 20.3? (It is ok to not updated test for those)

**Pull request merged**